asv_bench/asv.conf.json

```diff
@@ -70,12 +70,10 @@
     // the package name by 'pip+', and the package will be installed via
     // pip (with all the conda available packages installed first,
     // followed by the pip installed packages).
-    "matrix": {
-        "pandas": ["1.1.5"],
-        "packaging": [""],
-        "pip+ray": ["1.0.1"],
-        "pyarrow": ["1.0"]
-    },
+    // "matrix": {
+    //    "pip+ray": ["1.0.1"],
+    //    "pyarrow": ["1.0"]
+    // },
```

Comment:
what changed in this file?

Reply:
now dependencies are taken directly from the setup.py

Reply:
then why do we need to keep `ray` and `pyarrow` here?

Reply:
I left them as an example for the future.

Reply:
But I thought that the idea of taking dependencies directly from the `setup.py` is to get rid of pining them in asv config. Isn't this make us modify config all the time `ray` releases the new version?

Reply:
Yes, but if we need to test using the dependency matrix, then this is the only option even if we have to change the dependencies manually.

Reply:
Oh, I didn't notice that these lines commented)
Now I got the idea